src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/entrypoint/Entrypoint.tsx

```diff
@@ -44,31 +40,21 @@ export default function Entrypoint() {
           type={PresentationType.Stateless}
         >
           <div>
-            <Form/>
+            <Form />
           </div>
         </Presentation>
       );
     }
   }
 
   return (
```

Comment:
Maybe write some tests on the entrypoint.tsx to ensure the two return stated can happen?
Can be ass simple as `expect(wrapper.find('AltinnContentLoader')).toHaveLength(1)` and `expect(wrapper.find('Form')).toHaveLength(1)` with the right surrounding states?

Reply:
Good point. Fixing👍